src/dao/providers/base.ts

```diff
@@ -9,10 +10,12 @@ export interface ProviderUser {
 
 export interface ProviderActivity extends ActivityCreateOptions {}
 
+// I'm on the fence about the providers needing to know how to get auth information from the user model, but
+// it needs to live somewhere :(
```

Comment:
It get tough too when talking about maintaining the same interface for multiple providers. Some providers may not have an access key.

Reply:
ya I think that's why its best to leave it up to the provider implementation to know how to use the `User` to access the service.